gtsfm/utils/tracks.py

```diff
@@ -74,3 +77,28 @@ def classify_tracks3d_with_gt_cameras(
         tracks_2d.append(SfmTrack2d(measurements))
 
     return classify_tracks2d_with_gt_cameras(tracks_2d, cameras_gt, reproj_error_thresh_px)
+
+
+def get_triangulation_angle(track3d: SfmTrack, cameras: Dict[int, gtsfm_types.CAMERA_TYPE]) -> float:
```

Comment:
I think we want the `max` reflected in the name somehow `get_max_triangulation_angle` ?